helper/benchmark_output.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+    Benchmark Output Structure
+    This class serves as a data structure to contain the parsed (or not)
+    results of the benchmark, the dictionary of parsed perf results of running
+    the benchmark, and the metadata associated to this run of the benchmark
+
+"""
+
+
+class BenchmarkOutput(object):
```

Comment:
As per my comment above, this shouldn't be called "benchmark" output, but something generic, like "CommandOutput", so that all run methods can return it.